great_expectations/data_context/data_context/data_context.py

```diff
@@ -439,27 +438,8 @@ def delete_datasource(self, name: str) -> None:  # type: ignore[override]
         super().delete_datasource(datasource_name=name, save_changes=True)
         self._save_project_config()
 
-    def add_profiler(
```

Comment:
What's the explanation for this removal?
It's already on the ABC and we just forgot to remove it earlier?

Reply:
yup!